src/components/molecules/card/AnyWayCard.tsx

```diff
@@ -96,6 +292,13 @@ const AnyWayCard: FC<IProps> = ({
   const sizes = getSizes(variant, factor);
 
   const classes = useStyles();
+
+  const tooltipClasses = makeStyles(() => ({
+    tooltip: {
```

Comment:
there already a var of makeStyles in this file, it is called useStyles. you should use it instead of creating a new one. Try to copy the way it is already done, and talk to me if you need help